game/libda/fft.hpp

```diff
@@ -9,12 +9,11 @@
 #include <cstddef>
 #include <vector>
 
-extern const double m_pi;
-
-const double m_tau = (2.0 * m_pi);
 
 namespace da {
 
+	constexpr double TAU = 2.0 * 3.141592653589793238462643383279502884;
```

Comment:
Please define pi somewhere where all classes and namespaces can use it.
Now the code is 'cluttered' with 2 random pi digits. Effected place here and `game/util.hh#L8` als i wouldn't make it a fixed digit i'd calculate it like the following: `constexpr double pi() { return std::atan(1)*4; }`
`std::atan()` is defined in `#include <cmath>`